src/jit/flowgraph.cpp

```diff
@@ -21804,7 +21804,7 @@ Compiler::fgWalkResult      Compiler::fgUpdateInlineReturnExpressionPlaceHolder(
             else
             {
                 // Just assign the inlinee to a variable to keep it simple.
-                tree->CopyFrom(comp->fgAssignStructInlineeToVar(tree, retClsHnd), comp);
+                tree = comp->fgAssignStructInlineeToVar(tree, retClsHnd);
```

Comment:
Yes, the change was made for GT_RET_EXPR previously. It's not required now. Will undo it.